docs/src/examples/components/Alert/index.tsx

```diff
@@ -13,6 +14,7 @@ const AlertExamples = () => (
     <State />
     <Slots />
     <Rtl />
+    <Usage />
```

Comment:
`<Usage />` section should not go to `index.tsx` of any component example. The Usage is displayed under Usage tab.

Reply:
Oops, good catch 👍